sr_hand_kinematics/hand_kinematics/test/shadowhand_motor.srdf

```diff
@@ -0,0 +1,562 @@
+<?xml version="1.0" ?>
```

Comment:
Yes, I thought it might be better to have a copy here for the test as we may modify the one in the hand moveit config and then the tests will fail.